pyhap/accessory.py

```diff
@@ -165,6 +165,12 @@ def __init__(self, display_name, aid=None, mac=None, pincode=None,
 
         self._set_services()
 
+    def __repr__(self):
+        """Return the representation of the accessory."""
+        services = [s.display_name for s in self.services]
+        return "<accessory display_name='{}' services={}>" \
```

Comment:
Consistency using ticks around properties. Also I would write a capital A (for Accessories)